content/docs/guides/troubleshooting.md

```diff
@@ -17,6 +17,16 @@ Linux:
 grep -ci avx /proc/cpuinfo
 ```
 
+### AVX2
+Some components requirement AVX2 for better performance, e.g. TF Serving.
+To ensure the nodes support AVX2, we added minCpuPlatform arg in our deployment
+[config](https://github.com/kubeflow/kubeflow/blob/master/scripts/gke/deployment_manager_configs/cluster.jinja#L105).
+
+On GCP this will fail in regions (e.g. us-central1-a) that do not have Intel Haswell.
+In that case, please choose another region, or change the config to other 
+[platform](https://en.wikipedia.org/wiki/List_of_Intel_CPU_microarchitectures)
```

Comment:
How will using a newer platform fix this issue? If a region doesn't have haswell but it has a newer platform then wouldn't it already get scheduled there?

I thought mincpuplatform can be satisfied by newer platforms.

Reply:
No it needs to explicitly have that platform.

Reply:
Lets add a link to 
https://cloud.google.com/compute/docs/instances/specify-min-cpu-platform
as well